jest.js

```diff
@@ -0,0 +1,8 @@
+const swcJest = require("@swc/jest");
+const { config } = require("./lib/swc");
```

Comment:
The tests do not rely on `lib` currently, they rely on `src`. What happens if `lib` isn't present, will the tests require compilation now?

Reply:
I'd been using the `test` command which had been compiling before I realized that command was intended for the pipeline. This is a tricky one because ideally the register hook and the transformer share a config. However, if I convert them to ts files and move them to the top level then the TypeScript will generate a similar directory structure (files in `src` go to `lib/src` instead of just `lib`).

The simplest path, which is mentioned below, is to move the config to top level and make it a js file.